README.md

```diff
@@ -75,68 +78,101 @@ Use SnowCLI to build a function or stored procedure, or create a streamlit if yo
+1. Create a :file:`config.toml` file in that directory:
 
-   `vi config`
+   ```
+   $ cd ~/.snowflake
+   $ touch config.toml
+   ```
 
-3. Add a new configuration for your Snowflake connection with SnowCLI. You must prefix the configuration with `connections.`.
+1. In a text editor, open the :file:`config.toml` file for editing, such as the following for the Linux vi editor:
 
-For example, to add a Snowflake account `myaccount` for a user profile `johndoe` and a password of `hunter2`, add the following:
+   ```
+   $ vi config.toml
+   ```
 
-```ini
-[connections.connection_name]
-accountname = myaccount
-username = jondoe
-password = hunter2
-```
+1. Add a new Snowflake connection definition. You must prefix the configuration with **connections**.
 
-#### Add credentials manually using Windows
+   For example, to add a Snowflake connection called **myconnection** with the credentials: account **myaccount**,
+   user profile **johndoe**, and password **hunter2**,
+   add the following lines to the configuration file:
 
-1. Create a `.snowsql` folder with a `config` file at the following path: `%USERPROFILE%\.snowsql\config`
-2. Add a new configuration for your Snowflake connection with SnowCLI. You must prefix the configuration with `connections.`.
+   ```
+   [connections.myconnection]
+   accountname = myaccount
+   username = jondoe
+   password = hunter2
+   ```
 
-### Using environment variables
+1. If desired, you can add more connections, as shown:
 
-Environment variables can only be replaced in connection parameters. Parameter values in config file have to start with `$SNOWCLI_` and will be searched as a name in environment variables, but without `$` at the beginning.
+   ```
+   [connections.myconnection-test]
+   accountname = myaccount
+   username = jondoe-test
+    password = hunter2
+   ```
 
-For example connection config:
+1. Save changes to the file.
 
-```ini
-[connections.test]
-account = $SNOWCLI_ACCOUNT
-user = $SNOWCLI_USER
-password = $SNOWCLI_PASSWORD
-```
 
-will be replaced with environment variables:
+#### Use environment variables for Snowflake credentials
+
+If you prefer, you can specify Snowflake credentials in system environment variables, instead of specifying them
+in configuration files. You can use environment variables only to replace connection parameters. Parameter values in
+config file have to start with **\$SNOWCLI_**, which identify environment variables,
+but without **$** at the beginning.
+
+To add credentials as environment variables:
+
+1. Define the following environment variables, as appropriate for your operating system:
+
+   ```
+   SNOWCLI_ACCOUNT = "account"
+   SNOWCLI_USER = "user"
+   SNOWCLI_PASSWORD = "password"
+   ```
+
+
+1. If you already specify your credentials in a configuration file, as shown, you can remove them from the
+   connection settings.
+
+   ```
+   [connections.test]
+   account = $SNOWCLI_ACCOUNT
+   user = $SNOWCLI_USER
+   password = $SNOWCLI_PASSWORD
+   ```
```

Comment:
Today it work a bit different. If users have the following config:
```
key0 = value0

[section]
[section.option]
key1 = value1

[section2]
key2 = value2
```

They can override existing configuration by variables like:
```
SNOWFLAKE_KEY0
SNOWFLAKE_SECTION_OPTION_KEY1
SNOWFLAKE_SECTION2_KEY2
```

Reply:
Updated